hs_core/tests/api/native/test_show_in_discover.py

```diff
@@ -0,0 +1,92 @@
+import os
+
+from django.test import TestCase
+from django.contrib.auth.models import Group
+
+from hs_core import hydroshare
+
+
+class TestHideOldVersions(TestCase):
+    def setUp(self):
+        super(TestHideOldVersions, self).setUp()
+        self.group, _ = Group.objects.get_or_create(name='Hydroshare Author')
+
+        # create a user who is the owner of the resource to be versioned
+        self.owner = hydroshare.create_account(
+            'owner@gmail.edu',
+            username='owner',
+            first_name='owner_firstname',
+            last_name='owner_lastname',
+            superuser=False,
+            groups=[]
+        )
+        # create a generic resource
+        self.version0 = hydroshare.create_resource(
+            resource_type='GenericResource',
```

Comment:
I realize this is copy pasta from an older test, but this should be Composite.

Reply:
Fixed.